spec/presenters/client_summary_spec.rb

```diff
@@ -1,21 +1,26 @@
 describe ClientSummary do
-  it "adds status" do
+  it "adds non-canceled status" do
```

Comment:
what about `it "shows non-canceled proposals" do` -- does that fit the spec expectation more clearly?